include/GafferOSL/OSLCode.h

```diff
@@ -72,6 +72,8 @@ class OSLCode : public OSLShader
 		/// combo we use everywhere else.
 		ShaderCompiledSignal &shaderCompiledSignal();
 
+		virtual void loadShader( const std::string &shaderName, bool keepExistingValues=false );
```

Comment:
Can we add a comment describing why this is currently a no-op? I guess what would be really nice is if we could implement it so we populated the OSLCode node with the parameters and source from an existing .oso file, but as far as I know .oso files don't embed source. Definitely not something to worry about now...